### PR TITLE
Added directory structure schema for 3D IMC

### PR DESCRIPTION
**Pull request by icaoberg**
This add the directory structure YAML for 3D IMC given the [spec](https://docs.google.com/spreadsheets/d/1QnTsI0Brw1CUfQrwY_jul4u6E3pDk5I0hbJ886ZNSMU/edit#gid=1058624332) provided by @cebriggs7135

### CODE REVIEW

**Review by mccalluc — CHANGES_REQUESTED** [review]
Thanks: Lots of suggestions, but fixes should be easy.
- Style preference on the section headers: When I started from the top, I wasn't sure what `mcd` was -- adding a slash might make that more clear?
- Escape periods in regexes: Probably wouldn't actually cause problems, but better to be precise.
- PDFs: This is more of a policy question, but I'm thinking of this document as specifying something that latter contributors will have to adhere to, and I don't want to impose requirements on them which are unnecessary... and on the other side, I have preferred that the information be supplied in some form other than PDF, if it could be useful.

Stepping back from the details, at some point I would like to step back from the directory process, and if you say the structure is XYZ, it would be XYZ... I don't think we're there now, but if you feel I'm micromanaging things, please let me know, and I'll back off.

I'm going to checkout this branch and generate the documentation for it, so we can make sure that works.

File: src/ingest_validation_tools/directory-schemas/imc3d.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  pattern: '3D_image_stack.ome.tiff'
+  description: 'OME.tiff file comprising all slices and channels'
+  required: True
+  is_qa_qc: False
+
+-
+  pattern: 'extras'
+  description: 'Folder containing optional metadata'
+  required: False
+  is_qa_qc: False
```
[the remark this answers is not in the document]
Comment (icaoberg) [reply src/ingest_validation_tools/directory-schemas/imc3d.yaml:30]: @mccalluc got.

File: src/ingest_validation_tools/directory-schemas/imc3d.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  pattern: 'mapping/cluster_labels_image.tif'
+  description: 'Cell images labeled by cell type'
+  required: Yes
+  is_qa_qc: False
+
+#processed
+-
+  pattern: 'processed/umap_phenograph.pdf'
+  description: 'tSNE phenograph'
+  required: Yes
```
[the remark this answers is not in the document]
Comment (icaoberg) [reply src/ingest_validation_tools/directory-schemas/imc3d.yaml:51]: @mccalluc this is a question for @cebriggs7135 and @ropelews

**Review by jswelling — APPROVED** [review]

**Review by mccalluc — COMMENTED** [review]
I'll try wrap up the last part... there are merge conflicts with master.

**Comment by mccalluc** [comment]
@icaoberg : Question from earlier review has not been resolved:
>  `cells*organ*.csv`: There is no such csv, that I can see.
>  Do we know anything more about the structure of the file? If it's something automatically generated, I'd like to give the future user some sense of how it would be parsed.

**Pull request merged**